content/docs/04.5-devx/05-app-profile.md

```diff
@@ -65,9 +65,12 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: mysecret
-  namespace: mywebnamespace
+  namespace: {{.spectro.system.apptier.NAMESPACE}}
 data:
-  password: "{{.spectro.app.$appDepName-mongodb.PASSWORD}}"
+  
+  namespace: "{{.spectro.system.apptier.NAMESPACE}}" # Resolves to a value of the variable defined in App Profile tier parameters.yaml file.
+
+  password: "{{.spectro.app.$appDepName-mongodb.PASSWORD}}" # To refer the tier output variables of the top tiers.
```

Comment:
We should avoid adding comments in our code snippets as people often copy and paste them. We can move the information provided in the comment to above the code snippet.

Reply:
done